scylla-macros/src/parser.rs

```diff
@@ -35,6 +35,46 @@ pub(crate) fn parse_named_fields<'a>(
     }
 }
 
+pub(crate) enum StructFields<'a> {
+    Named(&'a FieldsNamed),
+    Unnamed(&'a FieldsUnnamed),
+}
+
+/// Parses the tokens_input to a DeriveInput and returns the struct name from which it derives and
+/// the fields
+pub(crate) fn parse_struct_fields<'a>(
+    input: &'a DeriveInput,
+    current_derive: &str,
```

Comment:
This is going to contain the trait name, not the struct name? Then I don't understand the docstring about the struct name.

Reply:
I adjusted the docstring comment.